src/services/notifications/create-push-notification-content.ts

```diff
@@ -26,17 +26,19 @@ export const createPushNotificationContent = ({
     { walletCurrency: baseCurrency },
   )
   const baseCurrencyName = baseCurrency === WalletCurrency.Btc ? "sats" : ""
-  const baseCurrencySymbol = baseCurrency === WalletCurrency.Usd ? "$" : ""
   const displayedBaseAmount =
     baseCurrency === WalletCurrency.Usd ? Number(amount.amount) / 100 : amount.amount
   const baseCurrencyAmount = displayedBaseAmount.toLocaleString(locale, {
+    minimumFractionDigits: 0,
     maximumFractionDigits: 2,
```

Comment:
Can maybe pass `MajorExponent.STANDARD` in here to be replaced later on with the selected display currency's major exponent

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/2399/commits/5ddccd2546ee3dd7dc47582be9898e4676329a19